src/point.py

```diff
@@ -0,0 +1,94 @@
+import math
+
+
+class Point:
+    """
+    A class for representing points on a two dimensional (2D) Cartesian plane.
+    """
+
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def distance_from_origin(self) -> float:
+        """
+        Calculate the Euclidean distance from this Point (self) and the origin (0, 0).
+
+        :return: The Euclidean distance from the self Point and the origin
+        :rtype: float
+        """
+        return math.sqrt(self.x**2 + self.y**2)
```

Comment:
Might be more instructive to `return self.distance_from_point(Point(0,0))`

Reply:
I went back and forth on that one, but seeing a 2nd-er for the idea, I'm game.